### ISSUE TITLE
Mail doesn't open - iOS - 11.2.2

### ISSUE THREAD

**Issue by lpotapczuk** [chirag04/react-native-mail#105]
Hi!

I have an issue with opening mail on the iOS. I have an iPhone X with 11.2.2. I have set up an email account on it. When I compose mail:

    Mailer.mail({
      subject: emailTitle,
      body: emailBody,
      isHTML: false,
    }, (error, event) => {
      console.log('Error while emailing notes', error, event);
    });

nothing happens. I can see the following logs in my debugger:

> 2018-04-06 15:24:14.365060+0200 example[271:9475] [MC] Filtering mail sheet accounts for bundle ID: com.example, source account management: 1

But then no window shows up. On Android everything is fine. Any ideas on whats going on?

**Comment by chirag04** [comment]
Are you running this on simulator? you need to run this on a real device.

**Comment by lpotapczuk** [comment]
Yes, I run this on the device. I found out that this was casued by invoking
setState method right after using Mailer. As of now I am using timeout to
open the mail view. :/

**Comment by punjasin** [comment]
same to me on ios 12

**Comment by jothikannan** [comment]
Same to me on IOS 12.1
